### ISSUE TITLE
add live dropdown search field for colleges

### ISSUE THREAD

**Issue by championswimmer** [coding-blocks/oneauth#424]
hard to scroll through the list
a searchable dropdown would be better.

@VibhorCodecianGupta  you can pick this up ?

**Comment by VibhorCodecianGupta** [comment]
@championswimmer yup, almost done. Could you open an issue for the todos of `organisations` model and architecture?

**Comment by arshadkazmi42** [comment]
@championswimmer I can take this up, if its still available?

**Comment by TdevM** [comment]
@arshadkazmi42 Yes. You can work on this.

**Comment by arshadkazmi42** [comment]
I got busy with someother work. 
If anyone else wants to work on this, Please go ahead.
